app/src/main/java/com/teamagam/gimelgimel/app/map/viewModel/MapViewModel.java

```diff
@@ -140,6 +146,11 @@ public void mapReady() {
         mDisplayMapEntitiesInteractor = mDisplayMapEntitiesInteractorFactory.create(this);
```

Comment:
ABORT.
I've looked at it again and wrote all my thoughts here/:
GGMap and its cesium implementation holds viewer camera and exposes an observable of that location.
It is currently used (subscribed) when the map view closes to save the last frustum so it can be reloaded later on (when the map displays again).

It is done that way because when the view stops, you cannot get the last frustum from the viewer, so it continually fetches while it's running so the frustum can be saved to the domain on the view's stop phase.

So there is no need for a displayer interface because it doesnt dispaly  anything.